.travis.yml

```diff
@@ -21,10 +21,10 @@ before_script:
 
 # run_tests.py runs everything under the script: tag so only put commands
 # under it that we want to run (and want to be able to run) as local tests
-script: 
+script:
   - flake8 *.py tests ga4gh scripts --exclude=ez_setup.py
   - nosetests --with-coverage --cover-package ga4gh
               --cover-inclusive --cover-min-percentage 85
-              --cover-branches --cover-erase
+              --cover-branches --cover-erase -v
```

Comment:
I did have it in during debug to solve problems where I had tests that locked up the run so I could figure out which one they were, but I've removed it now